packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -950,7 +950,50 @@ class Auth {
             })
         }
         this.clearStorage()
-        return await this.loginGuestUser()
+        return await this.ready()
```

Comment:
I switched this to `this.ready()` to take advantage of `queueRequest`. This limits (but does not completely eliminate) the amount of possible re-login requests that could be in flight when several SCAPI requests all fail with an invalid token. 

With this, I generally observe 2 in-flight login requests (rarely 3 but not more) before queueRequest returns the in-flight promise.

Reply:
After discussing this with @kevinxh , we agree that `queueRequest` as currently implemented is not able to handle parallel requests. I will add a separate ticket in our backlog for improving `queueRequest` to better handle parallel requests.

Reply:
A new ticket, W-17151903, was filed for updating the `queueRequest` implementation.